packages/manager/modules/pci/src/projects/project/storages/containers/translations/Messages_fr_FR.json

```
@@ -1,8 +1,15 @@
 {
   "pci_projects_project_storages_containers_archive_title": "Cloud Archive",
   "pci_projects_project_storages_containers_object_title": "Object Storage",
+  "pci_projects_project_storages_containers_object_description": "Hébergez votre catalogue de données dans notre solution Object Storage. Vous pouvez administrer ces données depuis votre application grâce aux API S3. Content !!!",
```

Hébergez votre catalogue de données dans notre solution Object Storage. Vous pouvez administrer ces données depuis votre application grâce aux API S3.
```suggestion
  "pci_projects_project_storages_containers_object_description": "Hébergez votre catalogue de données dans notre solution Object Storage. Vous pouvez administrer ces données depuis votre application grâce aux API S3.",
```